modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -341,6 +341,15 @@ private void exposePorts() {
         }
     }
 
+    @Override
+    protected void containerIsStarting(InspectContainerResponse containerInfo, boolean reused) {
+        if (reused) {
+            this.logger().debug("Couchbase container is being reused, assuming correct configuration.");
```

Comment:
`assuming correct configuration` doesn't sound right to me because the CouchbaseContainer implementation provides the right configuration.

I think the implementation should be

```java
if (!reused) {
    containerIsStarting(containerInfo);
}
```

```suggestion
           logger().debug("Couchbase container is being reused, assuming correct configuration.");
```